sympy/physics/control/lti.py

```diff
@@ -0,0 +1,553 @@
+            ((isinstance(den, Expr) and den.has(Symbol)) or den.is_number)):
+                obj = super(TransferFunction, cls).__new__(cls, num, den, var)
+                obj._num = num
+                obj._den = den
+                obj._var = var
+                return obj
+        else:
+            raise TypeError("Unsupported type for numerator or denominator of TransferFunction.")
+
+    @property
+    def num(self):
+        """Returns the Numerator of the Transfer Function.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> G1 = TransferFunction(s**2 + p*s + 3, s - 4, s)
+        >>> G1.num
+        p*s + s**2 + 3
+        >>> G2 = TransferFunction((p + 5)*(p - 3), (p - 3)*(p + 1), p)
+        >>> G2.num
+        (p - 3)*(p + 5)
+
+        """
+        return self._num
+
+    @property
+    def den(self):
+        """Returns the Denominator of the Transfer Function.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> G1 = TransferFunction(s + 4, p**3 - 2*p + 4, s)
+        >>> G1.den
+        p**3 - 2*p + 4
+        >>> G2 = TransferFunction(3, 4, s)
+        >>> G2.den
+        4
+
+        """
+        return self._den
+
+    @property
+    def var(self):
+        """Returns the variable that is used to anchor the symbols.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> G1 = TransferFunction(p**2 + 2*p + 4, p - 6, p)
+        >>> G1.var
+        p
+        >>> G2 = TransferFunction(0, s - 5, s)
+        >>> G2.var
+        s
+
+        """
+        return self._var
+
+    @property
+    def bound_symbols(self):
+        """Returns only variables that are dummy variables.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, l
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> G1 = TransferFunction(s + l, s - 2*p, s)
+        >>> G1.bound_symbols
+        [s]
+        >>> G2 = TransferFunction(p**2 + 2*p + 4, p - 6, p)
+        >>> G2.bound_symbols
+        [p]
+
+        """
+        return [self.var]
+
+    def _eval_subs(self, old, new):
+        arg_num = self.args[0].subs(old, new)
+        arg_den = self.args[1].subs(old, new)
+        argnew = TransferFunction(arg_num, arg_den, self.var)
+        return self if old in self.bound_symbols else argnew
+
+    def _eval_evalf(self, prec):
+        return TransferFunction(
+            self.num._eval_evalf(prec),
+            self.den._eval_evalf(prec),
+            self.var)
+
+    def _eval_simplify(self, **kwargs):
+        tf = cancel(Mul(self.num, 1/self.den)).as_numer_denom()
+        num_, den_ = tf[0], tf[1]
+        return TransferFunction(num_, den_, self.var)
```

Comment:
I have defined this method so that we are able to use sympy's own `simplify` function to cancel poles and zeros in a transfer function (which was not possible earlier). 
``` python
>>> G1 = TransferFunction((p + 3)*(p - 1), (p - 1)*(p + 5), p)
>>> from sympy import simplify
>>> simplify(G1)
TransferFunction(p + 3, p + 5, p)
>>> _ == G1.simplify()
True
```